test/validation/api/queue/queue.c

```diff
@@ -432,10 +431,12 @@ static void queue_test_info(void)
 	param.sched.prio = ODP_SCHED_PRIO_NORMAL;
 	param.sched.sync = ODP_SCHED_SYNC_ORDERED;
 	param.sched.group = ODP_SCHED_GROUP_ALL;
-	if (capability.max_ordered_locks)
+	if (capability.max_ordered_locks) {
 		param.sched.lock_count = 1;
-	else
+	} else {
+		printf("\n    Ordered locks NOT supported\n");
 		param.sched.lock_count = 0;
+	}
```

Comment:
Wouldn't it be simpler to say `param.sched.lock_count = capability.max_ordered_locks`?

Reply:
Will fix.